docker/Dockerfile.llm

```diff
@@ -0,0 +1,9 @@
+FROM pytorch/torchserve-nightly:latest-gpu as server
+
+USER root
+
+RUN mkdir /data && chown -R model-server /data
+
+USER model-server
+
+ENTRYPOINT [ "python", "-m", "ts.llm_launcher", "--vllm_engine.download_dir", "/data" ]
```

Comment:
Can we make this generic. What if we use a script and pass args . Ex: What do we do if we want to reuse this for TRT-LLM?

Reply:
plan is to use fire instead of arparse for the cli interface which will allow for arbitrary arguments and we can then select the engine and parse e.g. --vllm_engine.download_dir and --trt_llm.foo_bar based on the type.
Current interface will not be affected so we can add this in follow up prs.

Reply:
Sounds good!